Formula/textidote.rb

```diff
@@ -0,0 +1,55 @@
+class Textidote < Formula
+  desc "Spelling, grammar and style checking on LaTeX documents"
+  homepage "https://sylvainhalle.github.io/textidote"
+  url "https://github.com/sylvainhalle/textidote/archive/refs/tags/v0.8.3.tar.gz"
+  sha256 "8c55d6f6f35d51fb5b84e7dcc86a4041e06b3f92d6a919023dc332ba2effd584"
+  license "GPL-3.0-only"
+  head "https://github.com/sylvainhalle/textidote.git", branch: "master"
+
+  depends_on "ant" => :build
+  depends_on "openjdk"
+
+  def install
+    # Build the JAR
+    system "ant", "download-deps"
+    system "ant", "-Dbuild.targetjdk=1.7"
+
+    # Install the JAR + a wrapper script
+    libexec.mkpath
+    libexec.install "textidote.jar"
+    bin.mkpath
+    bin.write_jar_script libexec/"textidote.jar", "textidote"
+
+    # Install the completions script
+    bash_completion.mkpath
```

Comment:
This isn't needed, brew should do this automatically

Reply:
This is solved by another suggestion.